<launch>
    <!-- This version uses the openai_ros environments -->
    <rosparam command="load" file="$(find turtlebot_openai)/config/turtlebot_openai_qlearn_params.yaml" />
    <!-- Launch the training system -->
    <node pkg="turtlebot_openai" name="turtlebot_qlearn" type="start_qlearning.py" output="screen"/>
</launch>